packages/p/plasma-workspace-wallpapers/package.yml

```diff
@@ -0,0 +1,22 @@
+name       : plasma-workspace-wallpapers
+version    : 5.27.10
+release    : 1
+source     :
+    - https://download.kde.org/stable/plasma/5.27.10/plasma-workspace-wallpapers-5.27.10.tar.xz : 82862d512993b958f7f96e191ce951ff2748012292941f73cdd308f37e2e64d0
+homepage   : https://kde.org/plasma-desktop
+license    :
+    - GPL-2.0
```

Comment:
Licenses should use a valid SPDX identifier. Example if it is GPL 2.0 or later its: GPL-2.0-or-later https://spdx.org/licenses/